iperf version 2.0.4 running in server mode exits when a client connects
using the -d (duplex) flag.

To reproduce: On machine foo, run server:

[somlo@foo]$ iperf -s

on machine bar, run client in duplex mode:

[somlo@bar]$ iperf -c foo -d


What happens: At the end of the test, the iperf process on machine foo exits.

What *should* happen: At the end of the test, the iperf process on machine
should keep running, listening for new connections.

===

Patch form L. Gabriel Somlo on the iperf maillist 2009-01-21
More precisely: http://sourceforge.net/tracker/index.php?func=detail&aid=1983829&group_id=128336&atid=711371

Signed-off-by: Wolfram Sang <w.sang@pengutronix.de>

---
 src/Client.cpp |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

Index: src/Client.cpp
===================================================================
--- src/Client.cpp.orig
+++ src/Client.cpp
@@ -207,10 +207,10 @@ void Client::Run( void ) {
     char* readAt = mBuf;
 
 #if HAVE_THREAD
-    if ( !isUDP( mSettings ) ) {
-	RunTCP();
-	return;
-    }
+//     if ( !isUDP( mSettings ) ) {
+// 	RunTCP();
+// 	return;
+//     }
 #endif
     
     // Indicates if the stream is readable 
